bin/console

```diff
@@ -1,14 +1,147 @@
 #!/usr/bin/env ruby
+# frozen_string_literal: true
 
-require "bundler/setup"
-require "ibm/cloud/sdk"
+require 'bundler/setup'
+require 'ibm'
+require 'pry'
+require 'thor'
+require 'rest-client'
 
-# You can add fixtures and/or initialization code here to make experimenting
-# with your gem easier. You can also use a different console, if you like.
+# Encapsulate the API entryponts.
+class Sdk
```

Comment:
This is definitely interesting, can we investigate this in its own PR?

Reply:
Sure, how do I split these into their own PRs? Although I created this a couple of weeks ago and with CloudSDK I think it may become redundant.

Reply:
I like to split up a branch a couple of different ways depending on how involved the number of commits are.

If you have one or two commits that you want to pull out into a new branch you can do e.g.
```
git checkout -b new_pr_branch master
git cherry-pick sha-from-other-branch
```

If there are a lot of commits you can make a copy of your current branch and `git rebase` to drop unrelated commits.

Reply:
I just noticed this pr is off of your fork's master so `git checkout -b new_pr_branch master` won't work as expected.  You can `git checkout -b new_pr_branch upstream/master` where `upstream` is whatever you named your remote for this repo